cmd/ocitools/generate.go

```diff
@@ -366,19 +360,12 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 	return nil
 }
 
-func checkNs(nsMaps map[string]string, nsName string) bool {
-	if _, ok := nsMaps[nsName]; !ok {
-		return false
-	}
-	return true
-}
-
-func setupLinuxNamespaces(g *generate.Generator, needsNewUser bool, nsMaps map[string]string) {
+func setupLinuxNamespaces(g *generate.Generator, needsNewUser bool, context *cli.Context) {
```

Comment:
I'd rather have [`context` first][1], but that isn't a big deal ;).

[1]: https://github.com/opencontainers/ocitools/pull/159#discussion_r72564169